containers/hugging-face-inference/README.md

```diff
@@ -0,0 +1,33 @@
+# Hugging Face Models
+
+### Deploy models in Serverless Containers
```

Comment:
**issue(structure):** our examples should all use the standard README format included in the top-level of the repo.